subsys/fs/CMakeLists.txt

```diff
@@ -16,6 +16,16 @@ if(CONFIG_FILE_SYSTEM)
   target_link_libraries_ifdef(CONFIG_FAT_FILESYSTEM_ELM   FS INTERFACE ELMFAT)
   target_link_libraries_ifdef(CONFIG_FILE_SYSTEM_LITTLEFS FS INTERFACE LITTLEFS)
   target_link_libraries_ifdef(CONFIG_FILE_SYSTEM_NFFS     FS INTERFACE NFFS)
+
+  if (CONFIG_FILE_SYSTEM_NFFS)
```

Comment:
If multiple filesystems should be supported as stated in the comment below, I would say this `if` should be removed.

Reply:
agree